### ISSUE TITLE
Feature Request - Separate volume control for alarms

### ISSUE THREAD

**Issue by wbrco** [gazugafan/RestStop#4]
Can a volume adjustment be added to the alarm sounds? 

User Story:
As a person that uses RestStop as well as video/audio conferencing, I'd like to be able to set the alarm volume lower so it doesn't distract me during calls/meetings.

Bonus if I can just right mouse click on the hourglass and there's a Volume listing instead of having to open the Settings.  :)

**Comment by gazugafan** [comment]
Makes sense! There's no way to do this currently, but a workaround would be to change the volume of the actual sound file. You could do this in something like Audacity, or maybe use something like https://mp3cut.net/change-volume (untested).
